coinjoins/README.md

```diff
@@ -0,0 +1,79 @@
+# Coinjoins & Privacy UX Primer
+
+- Bitcoin is not anonymous.
+- Consider each address you generate as a new identity.
+- Coin has history
+- Histories can become linked
+- Public ledger
+- Anyone can lookup
+- New users may not be aware
+
+## Address Reuse
+
+Use a [hierarchical deterministic wallet](https://en.bitcoin.it/wiki/Deterministic_wallet) in order to avoid reusing an identity.
+
+Practicality
+
+- How do you adverise an address to be paid in a video?
+
+## Wallet Fingerprinting
+
+The way the transaction is constructed can give chain analysts an idea of which wallet was used.
+
+1. Change is sometimes the last output
+   1. Solutions
+      1. Lexicographical Ordering
+2. If the timelock in the transaction is a specific amount
+
+## Routing Transaction
+
+- Should use Tor
+  - Research Dandelion
+- If an attacker has enough nodes in the network it can determine the location of where the transaction was broadcasted.
```

Comment:
rebroadcast attacks also interact with dust attacks here, by dusting old addresses spy nodes that aggressively connect can increase likelyhood of broadcast being linkable via network level information.

Cc @amitiuttarwar - see [twitter search for talks, PR reviews on rebroadcast work](https://twitter.com/search?q=amizi%20rebroadcast&src=typed_query)

and also block lists (e.g. AWS IP ranges etc), a number have circulated